chat-client/src/client/mynahUi.ts

```diff
@@ -692,9 +707,12 @@ ${params.message}`,
             commands: toContextCommands(group.commands),
         }))
 
+        console.log('Context commands feature ', featureConfig?.get('highlightCommands'))
+        const contextCommands = [...(contextCommandGroups || []), ...(featureConfig?.get('highlightCommands') || [])]
+
         Object.keys(mynahUi.getAllTabs()).forEach(tabId => {
             mynahUi.updateStore(tabId, {
-                contextCommands: contextCommandGroups,
+                contextCommands: contextCommands,
```

Comment:
This will not work: `contextCommands` do not have `variation` and `value.string..` properties.